### PR TITLE
Dev_nath 

### PR DESCRIPTION
**Pull request by NateThibault**
J'aimerais merge sur ma branche main (Student_nath). Mon projet est complet et il va rester quelques features à ajouter selon mes critères.

Merci

### CODE REVIEW

File: apropos.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<html>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<head>
+    <title>À Propos</title>
```
Comment (WilliamZimmermann) [inline apropos.html:5]: I think that a better title would be "About".

Comment (NateThibault) [reply apropos.html:5]: Good Thanks !

**Review by WilliamBitton — APPROVED** [review]
Well-organized and easy-to-understand CSS code.
HTML code made by following bootstrap templates.
Bootstrap was used when possible instead of manual CSS.
Inline CSS style tag was used as little as possible.
Code well structured, header main footer.
No code repetition.
Tags are well used.

File: apropos.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
+<html>
```
Comment (WilliamZimmermann) [inline apropos.html:1]: Try to maintain file names, as well components and properties, in English. The content can be in French, no problem. So, for this case, `about.html`.

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  text-decoration: none !important;
+  color: grey;
+  cursor:default
+}
+@media only screen and (max-width: 915px) {
+  .text-decoration-accueil{
+    font-size: 105% ;
+  }
+}
+.homeIconIndex{
```
Comment (WilliamZimmermann) [inline css/styles.css:80]: I noticed that sometimes you are using pascal-case, other kebab-case and others camelCase. Try to follow the same standard.

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+  margin-left: -20px;
+}
+@media only screen and (max-width: 915px) {
+  .homeIconIndex{
+    width:15px !important;
+    height:15px !important;
+    margin-bottom: 1px;
+  }
+}
+.homeIcon{
```
Comment (WilliamZimmermann) [inline css/styles.css:93]: here too

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+  }
+}
+
+
+/* CSS POUR LE FORMULAIRE CONTACT*/
+.form input{
+  height: 6%;
+  border-radius: 5px;
+}
+.reseauSocial p{
```
Comment (WilliamZimmermann) [inline css/styles.css]: here too. Also, try to use class names in english.

File: css/styles.css [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+  margin-left: -30px;
+  font-size:x-large
+}
+.reseauSocial li{
```
Comment (WilliamZimmermann) [inline css/styles.css]: Take a look in all the file to see if you have more differences and try to standardize it.

File: projets.html [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+<head>
+    <title>Projets</title>
+    <link rel="stylesheet" href="css/bootstrap.min.css">  
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/ionicons/2.0.1/css/ionicons.min.css">
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+<body style="background-color: #f1f1f1;">
```
Comment (WilliamZimmermann) [inline projets.html]: Try to put styles in your `css/style.css` file.

File: projets.html [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+                            <p class="card-text text-justify p-3">Mon portfolio web est un site internet professionnel qui me permet de présenter mon parcours, mes compétences et mes réalisations de manière claire et concise. Il s'adresse à un public varié, tel que des employeurs potentiels, des clients, des partenaires ou des collaborateurs. Il contient une page de présentation , une page de mes projets réalisés et une page avec un formulaire pour me joindre par courriel.</p>
+                        </div>
+                    </div>
+                </div>
+            </div>    
+        </div>
+    </main>
+    <footer class="footer-basic">
+        <div class="social">
+            <a href="https://github.com/NateThibault"><img src="assets/github.svg" width="30px" height="30px"></a>
```
Comment (WilliamZimmermann) [inline projets.html]: Try to use `target='_blank'` instead to open in the same page. The same for the next external links.

**Review by WilliamZimmermann — APPROVED** [review]
Très bon travail. Quelques commentaires pour améliorer votre code. Cependant, j'ai vu un code très bien développé. Félicitations !

File: apropos.html (hunk 2 above)
Comment (NateThibault) [reply apropos.html:1]: Jai corrigé le nom pour about.html

File: projets.html (hunk 8 above)
Comment (NateThibault) [reply projets.html]: Les corrections ont été appliquées

File: projets.html (hunk 7 above)
Comment (NateThibault) [reply projets.html]: C'est corrigé !

File: css/styles.css (hunk 6 above)
Comment (NateThibault) [reply css/styles.css]: C'est corrigé pour socialNetwork

**Pull request merged**